from __future__ import annotations


class HaltError(Exception):
    pass
